joblib/test/test_memory.py

```diff
@@ -421,6 +421,10 @@ def test_persistence(tmpdir):
     memory2 = pickle.loads(pickle.dumps(memory))
     assert memory.store_backend.location == memory2.store_backend.location
 
+    # Fix regression in #741
```

Comment:
Maybe use the issue full URL (https://github.com/joblib/joblib/issues/741) instead of just the id ?